data-raw/admiral_verify_templates.R

```diff
@@ -0,0 +1,238 @@
+# This script:  data-raw/admiral_verify_templates.R
+
+#
+# Generates ADaM from templates and compares to previously generated ADaM file (found in pharmaverseadam).
+# pharmaverseadam is the SOURCE.
+#
+# Much code taken from pharamavreseadam::create_adams_data.R
```

Comment:
Keep?

Reply:
What does FIX mean?  

I think a purpose is helpful for someone looking at this script

Reply:
@bms63  
-  still many comments in code, notes (like `FIX` or `DISCUSS`) that will be removed/cleaned.   
- @StefanThoma spent over an hour with me today ... and much progress made, specifics nailed down...
- Once today's changes are made (Thurs?) expect only usually minor fixes.
- You run code NOW to see resulting databases `run verify_templates()` but about to make today's changes.
- There are several directories in use (old AdAm, new ADaM, cache,  results from diffdf ...), some  only for coding convenience.  But please leave till we are all happy with results.  Then can add `cleanup`.
- As to workflow, have basic one to run R scripts.  Stefan is backstop if I get stuck.